content/post/proxysql-webinar-2023.02.01/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: MariaDB and Percona-XtraDB Support for KubeDB ProxySQL, ProxySQL Dashboard & Alerts
+date: "2023-02-01"
+weight: 20
+authors:
+- Tasdidur Rahman
+tags:
+- kubedb
+- kubernetes
+- mysql
+- mariadb
+- percona-xtradb
+- provisioner
+- proxysql
+---
+
+## Summary
+
+On FEB 1st, 2023, AppsCode held a webinar on **"MariaDB and Percona-XtraDB Support for KubeDB ProxySQL, ProxySQL Dashboard & Alerts"**. <br>
+The essential contents fo this webinar are: <br>
```

Comment:
The essential contents for this webinar are

Reply:
done